generator/recipe.yaml

```diff
@@ -2,14 +2,14 @@ name: {{.Name}}
 source:
 {{- range $key, $value := .Source }}
   type: {{$key}}
-  config: {{$value}}
+  config: {{$value | indent 4}}
```

Comment:
this produces 5 indentations when generating bigquery extractor. This is not due to the logic of the template or the indent function, but it is because the `sampleConfig` in `bigquery` [extractor](https://github.com/odpf/meteor/blob/main/plugins/extractors/bigquery/bigquery.go#L40) has a space before each config.

can you help to make sure all of our sample configs does not have interfering whitespace on its own except the first new line?